pkg/api/secretgenerator/testdata/zz_fixture_TestLoadConfigFromPath_single_parameter_with_multiple_values.yaml

```diff
@@ -1,7 +1,4 @@
-- attachments:
-  - cmd: echo -n Attachment1
-    name: Attachment1
-  fields:
+- fields:
```

Comment:
I think I would feel more comfortable if we didn't change any existing fixtures to proof that the current as we have it right now will continue to work (only add some more to show that the new config will also work)